air/src/execution_step/lambda_applier/applier.rs

```diff
@@ -77,26 +93,68 @@ fn select_by_path_from_stream<'value>(
         }
         ValueAccessor::Error => unreachable!("should not execute if parsing succeeded. QED."),
     };
-
+    let idx = idx as usize;
     let stream_size = stream.len();
     let value = lambda_to_execution_error!(stream
         .peekable()
-        .nth(idx as usize)
+        .nth(idx)
         .ok_or(LambdaError::CanonStreamNotHaveEnoughValues { stream_size, idx }))?;
 
     let result = select_by_path_from_scalar(value, body.iter(), exec_ctx)?;
-    let select_result = StreamSelectResult::from_cow(result, idx);
+    let select_result = LambdaResult::from_cow(result, idx);
     Ok(select_result)
 }
 
+fn select_by_path_from_canon_map<'value>(
+    canon_map: &'value CanonStreamMap<'_>,
+    lambda: &NonEmpty<ValueAccessor<'_>>,
+    exec_ctx: &ExecutionCtx<'_>,
+) -> ExecutionResult<Cow<'value, JValue>> {
+    let (prefix, body) = lambda.split_first();
+
+    // HashMap<'map>::get(key: &'key K) forces key's lifetime 'key to be as good as 'map.
+    // This variance-derived requirement forces StreamMapKey<'static> here.
+    // See https://github.com/rust-lang/rust/issues/80389#issuecomment-752067798qwe
+    // for the details.
+    let stream_map_key: StreamMapKey<'static> = match prefix {
```

Comment:
It works for me if I replace `'static` with `'_`.

Reply:
Fixed.